static/chart.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Google Chart</title>
+  
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"></script>
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+  <link href="http://getbootstrap.com/dist/css/bootstrap.min.css" rel="stylesheet" />
+  <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+      <script type="text/javascript">
+      google.load('visualization', '1.0', {'packages':['corechart']});
+      google.setOnLoadCallback(drawChart);
+
+      function drawChart() {
+      var qstr = window.location.search;
+       // alert(qstr);
+         
+        qstr=qstr.substring(1);
+        //alert(qstr);
+        var params = {};  
+        var arr = qstr.split('&');
+        for(var i=0; i<arr.length;i++){
+          var vals= arr[i].split('=');
+
+          params[vals[0]] = vals[1];
+        }
+          //alert(params.chse);
+        var arr = [
+                ['Mushrooms', 0]
+              /*  ['Onions', 1],
+                ['Olives', 1],
+                ['Zucchini', 1],
+                ['Pepperoni', 2]*/
+              ];
+        if(params.chse == '1') {
```

Comment:
try to make a list of these properties, loop through them and add them with something like arr.push['chese', params.chse] (if parmas.chse is either 1 or undefined, you can use params.chse?1:0)